docs/docs/what-is-flockmtl.md

```diff
@@ -21,20 +17,32 @@ import TOCInline from '@theme/TOCInline';
-    - [`llm_complete_json`](/docs/semantic-functions/scalar-functions/llm-complete-json): Produces JSON-formatted text completions.
-    - [`llm_filter`](/docs/semantic-functions/scalar-functions/llm-filter): Filters data based on language model evaluations, returning boolean values.
-    - [`llm_embedding`](/docs/semantic-functions/scalar-functions/llm-embedding): Generates embeddings for input text, useful for semantic similarity tasks.
-    - [`fusion functions`](/docs/semantic-functions/scalar-functions/fusion-functions): Combines results from multiple queries based on relative relevance.
+    - [`llm_complete`](/docs/scalar-functions/llm-complete): Generates text completions using a specified language model.
+    - [`llm_complete_json`](/docs/scalar-functions/llm-complete-json): Produces JSON-formatted text completions.
+    - [`llm_filter`](/docs/scalar-functions/llm-filter): Filters data based on language model evaluations, returning boolean values.
+    - [`llm_embedding`](/docs/scalar-functions/llm-embedding): Generates embeddings for input text, useful for semantic similarity tasks.
 
 - **Aggregate Reduce Functions**:
-    - [`llm_reduce`](/docs/semantic-functions/aggregate-functions/llm-reduce): Aggregates multiple inputs into a single output using a language model.
-    - [`llm_reduce_json`](/docs/semantic-functions/aggregate-functions/llm-reduce-json): Similar to `llm_reduce`, but outputs JSON-formatted results.
-    - [`llm_rerank`](/docs/semantic-functions/aggregate-functions/llm-rerank): Reorders query results based on relevance scores from a language model.
-    - [`llm_first`](/docs/semantic-functions/aggregate-functions/llm-first): Selects the top-ranked result after reranking.
-    - [`llm_last`](/docs/semantic-functions/aggregate-functions/llm-last): Selects the bottom-ranked result after reranking.
+    - [`llm_reduce`](/docs/aggregate-functions/llm-reduce): Aggregates multiple inputs into a single output using a language model.
+    - [`llm_reduce_json`](/docs/aggregate-functions/llm-reduce-json): Similar to `llm_reduce`, but outputs JSON-formatted results.
+    - [`llm_rerank`](/docs/aggregate-functions/llm-rerank): Reorders query results based on relevance scores from a language model.
+    - [`llm_first`](/docs/aggregate-functions/llm-first): Selects the top-ranked result after reranking.
+    - [`llm_last`](/docs/aggregate-functions/llm-last): Selects the bottom-ranked result after reranking.
+
+
+This allows users to perform tasks such as text generation, summarization, classification, filtering, fusion, and embedding generation.
+
+## Hybrid Search Functions
+
+FlockMTL also provides hybrid search functions that combine vector and semantic search capabilities. These functions allow users to perform complex queries that leverage both traditional SQL operations and advanced language model features.
```

Comment:
`FlockMTL also provides functions that support hybrid search. Namely, the following data fusion algorithms to combine scores of various retrievers:`